src/bp-forums/forums/functions.php

```diff
@@ -2036,7 +2036,7 @@ function bbp_pre_get_posts_normalize_forum_visibility( $posts_query = null ) {
  *                        and forum id
  */
 function bbp_forum_query_topic_ids( $forum_id ) {
-	$topic_ids = bbp_get_public_child_ids( $forum_id, bbp_get_topic_post_type() );
+	$topic_ids = bbp_get_public_child_ids( $forum_id, bbp_get_topic_post_type(), true );
```

Comment:
@meetmakadiya If we always passing `$force_cache= true` then I don't think cache will useful. instead of this, we need to find action on which cache should be flush.

Reply:
@dipskakadiya 
Yes, that's correct this only executing on Post status update.
Thanks